deploy/gallery/instructions.md

```diff
@@ -55,5 +57,16 @@ The instructions on this page will guide you through configuring the pipeline te
     | 8|EnvCode | \<environmentCode> | Same as used in the deployment steps|
     | 9|KeyVaultName | kvp<10-character-random-string>| Get the name from \<environmentCode>-pipeline-rg |
 
-5. All set to start the pipeline now. Press **Debug**.
+4. All set to start the pipeline now. Click **Debug**.
+
+# Verify Success
+After the pipeline has run, you should see an indication of Success for the pipeline in the Synapse Workspace. Also, you can find the geojson file that was generated by doing the following steps:
```

Comment:
that was generated using the following steps

Reply:
made the suggested change